manifests/cf-manifest/deployments/040-cf-properties.yml

```diff
@@ -615,3 +615,290 @@ properties:
         client_key: (( grab secrets.bbs_client_key ))
         require_ssl: true
       traffic_controller_url: (( grab properties.loggregator.traffic_controller_url ))
+    rootfs_cflinuxfs2:
+      trusted_certs: |
+        -----BEGIN CERTIFICATE-----
```

Comment:
Do you think it would be easier to maintain this, especially if we add other CAs in the future, if it was a separate property/file that could be named (e.g. `aws_rds_certificates`)?